src.ts/agent/config.ts

```diff
@@ -191,25 +193,24 @@ export interface DebugAgentConfig {
    * @private
    */
   internal: {
-    registerDelayOnFetcherErrorSec: number,
-    maxRegistrationRetryDelay: number
-  },
+    registerDelayOnFetcherErrorSec: number; maxRegistrationRetryDelay: number;
```

Comment:
nit: any way to make clang-format comply with a newline between these two lines?